app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt

```diff
@@ -55,6 +55,14 @@ class PinPasswordActivityPresenter @Inject constructor(
       (activity as PinPasswordActivity).finish()
     }
 
+    binding.apply {
+      pinPasswordInputPinEditText.setOnClickListener {
+        pinPasswordInputPinEditText.isFocusable = true
+        pinPasswordInputPinEditText.isFocusableInTouchMode = true
+        pinPasswordInputPinEditText.requestFocus()
+      }
+    }
```

Comment:
```suggestion
    binding.pinPasswordInputPinEditText.setOnClickListener { pinEditText ->
        pinEditText.isFocusable = true
        pinEditText.requestFocus()
      }
```

You have already set pinEditText.isFocusableInTouchMode = true in xml.

Reply:
Got it.Thank you